WordPress/Classes/Networking/NotificationsServiceRemote.swift

```diff
@@ -8,6 +8,16 @@ import UIDeviceIdentifier
 ///
 public class NotificationsServiceRemote : ServiceRemoteWordPressComREST
 {
+    #if DEBUG
+        static let WordPressComApiPushAppId = "org.wordpress.appstore.dev"
+    #else
+    #if INTERNAL_BUILD
+        static let  WordPressComApiPushAppId = "org.wordpress.internal"
+    #else
+        static let WordPressComApiPushAppId = "org.wordpress.appstore"
+    #endif
+    #endif
```

Comment:
Does any other part of the code needs to know about this settings? Or is this something that will need to change in the future?